arch/arm64/core/reset.S

```diff
@@ -60,7 +60,7 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__reset_prep_c)
 2:
 	/* Disable alignment fault checking, disable D-Cache, I-Cache, MMU for safety */
 	mrs	x0, sctlr_el2
-	mov	x1, #(SCTLR_A_BIT | SCTLR_C_BIT | SCTLR_M_BIT | SCTLR_I_BIT)
```

Comment:
This sort of reverts 573a712bed06d984aab4a0b7424fc146e72c0921 but keeps less efficient code (2 instr instead of 1) and gets the comment 2lines above out of sync.

I think you can revert the commit completely to get back to the previous state of the instrs & comment, to then add your alternative solution in reset.c.

Reply:
Revert the old one, thanks.